Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/Views.kt

```diff
@@ -74,3 +85,169 @@ fun RecyclerView.onScroll(block: (Boolean) -> Unit) {
         }
     )
 }
+
+/**
+ * On [RecyclerView] item swipe listener
+ * @param context [Context]
+ * @param excludedPositions excluded positions from swiping factory
+ * @param onSwipe on swipe callback. It passes item's position and swipe direction
+ *
+ * Usage:
+ * ```
+ * RecyclerView.onSwipeItem(
+ *   context = requireContext(),
+ *   excludedPositions = listOf(0, 1)
+ * ) { position, direction ->
+ *   // Do operation here
+ * }
+ * ```
+ */
+fun RecyclerView.onSwipeItem(
```

Comment:
Lot's of  code in a very generic `Views.kt` class, should we maybe put it in a `util.lists.SwipeExtension`?